android/src/main/java/com/reactnativecommunity/androidprogressbar/ReactProgressBarViewManager.java

```diff
@@ -62,26 +105,31 @@ protected ProgressBarContainerView createViewInstance(ThemedReactContext context
     return new ProgressBarContainerView(context);
   }
 
+  @Override
   @ReactProp(name = PROP_STYLE)
-  public void setStyle(ProgressBarContainerView view, @Nullable String styleName) {
+  public void setStyleAttr(ProgressBarContainerView view, @Nullable String styleName) {
     view.setStyle(styleName);
   }
 
-  @ReactProp(name = ViewProps.COLOR, customType = "Color")
+  @Override
+  @ReactProp(name = ViewProps.COLOR)
```

Comment:
Why is `customType = "Color"` gone here?

Reply:
Because of adding `processColor` in JS the prop stopped working when `customType = "Color"` was there.

Reply:
Hmm looking e.g. here: https://github.com/software-mansion/react-native-svg/blob/1126079425a1b1a6c6094434f1c199c52ded1817/src/fabric/AndroidSvgViewNativeComponent.ts#L33 and then here: https://github.com/software-mansion/react-native-svg/blob/18a2f93e4057bae3dbdb604d9590861450f23b11/android/src/main/java/com/horcrux/svg/SvgViewManager.java#L101 and here: https://github.com/software-mansion/react-native-svg/blob/18a2f93e4057bae3dbdb604d9590861450f23b11/android/src/main/java/com/horcrux/svg/SvgViewManager.java#L101 and here: https://github.com/software-mansion/react-native-svg/blob/1126079425a1b1a6c6094434f1c199c52ded1817/src/elements/Svg.tsx#L179, maybe we should not use `processColor` in JS but just do something similar to how it is done in svg?

Reply:
Why was `processColor` added in the first place?

Reply:
Done in https://github.com/react-native-progress-view/progress-bar-android/pull/90/commits/45c5d5ec85f3319e8f866e1444574a14e62fa1e3. I don't particularly remember why I've done it this way 😅.